include/parsed_data_out.h

```diff
@@ -23,7 +23,8 @@ class ParsedDataOut : public ParameterAcceptor
       ex "solution_" produces solution_001, solution_002, etc.. . */
   ParsedDataOut (const std::string &name="",
                  const std::string &default_format="vtu",
-                 const std::string &run_dir="",
+                 const std::string &run_dir_input="",
```

Comment:
Can we go back to the name we came up with last time? `run_dir_input` is not really descriptive...

`incremental_run_prefix`

?

Also, can you document what it does?